components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -105,3 +105,62 @@ impl Manifest {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_manifest_generation() {
+        let mut m = Manifest {
+            metadata_name: "hi-f84iq".to_owned(),
+            habitat_name: "hi".to_owned(),
+            image: "core/hi".to_owned(),
+            count: 3,
+            service_topology: Default::default(),
```

Comment:
Wouldn't `Topology::default()` be clearer?

Reply:
from the docs, it seems Default::default() is prefered as type is in the declaration anyway.